src/web/screens/manageParticipants.tsx

```diff
@@ -33,7 +35,7 @@ function ManageParticipants() {
   };
 
   const reloader = useRevalidator();
-  const onParticipantRequestsUpdate = useCallback(() => {
+  const onParticipantUpdate = useCallback(() => {
```

Comment:
Maybe we could give this a better name, having both `onUpdateParticipant` and `onParticipantUpdate` will be confusing. We could call this `onParticipantUpdated` or `handleParticipantUpdated`, per the other uses of `reloader.revalidate()` in our codebase.